leandroalonso/WordPress-iOS
# Update card design and do not display author name when "Me" is selected

Ref https://github.com/wordpress-mobile/WordPress-iOS/issues/11646

In this PR:

* The post card was redesigned;
* The author is not displayed when "Me" is selected as a filter;
* Fixed a bug that was displaying a large empty area at the top of the posts lists

To test:

Use the blueprint in the master issue as a starting point. These are some user-scenarios that can be tested:

* Check how Published, Drafted, Scheduled and Trashed posts look
* Make sure that when selecting "Me" your name is *not* displayed
* Trash some posts, then Undo
* Edit posts and check that they update
* Test in larger devices (ie.: iPad) as well with increased font sizes
* Upload media and check that the progress bar is correctly displayed
* Remove your internet connection, change a post and make sure that a "Retry" button is displayed

Here are some screenshots:

![post_list_redesign](https://user-images.githubusercontent.com/7040243/57743430-6cdde280-769b-11e9-99db-1365b01223bc.png)

Update release notes:
